README.md

```diff
@@ -1,2 +1,4 @@
-# c_calculator
-calculator made using C
+# cpp_calculator
+C calculator thing but rewritten in c++ to allow of usage of c++ standard library instead of using outdated language
```

Comment:
change wording, as C isnt fully outdated, just fairly low level

Reply:
c is outdated and you can not change my mind 
![foxsockangry](https://user-images.githubusercontent.com/36876492/180379166-aca44671-6f00-4aaf-a852-34c0ccdb5576.png)